### PR TITLE
add MSSQL driver

### PR DESCRIPTION
**Pull request by tingwei628**
As title, I added  [MSSQL driver](https://github.com/denisenkom/go-mssqldb).

Thanks!

### CODE REVIEW

**Comment by imiric** [comment]
Hi @tingwei628, thanks for your contribution and terribly sorry for the extremely late response. :disappointed: I missed the GitHub notification and don't check this project often.

We'd definitely like to merge this. Would you mind fixing the merge conflicts and maybe adding a `test.js` for it? I'd like to improve the testing situation for this extension so that we could have integration tests for all supported RDBMSs, and run them in CI. But in the meantime moving the existing `test.js` for SQLite into a `tests` directory and adding the MSSQL variant would improve the situation. There is an [official SQL Server Docker image](https://hub.docker.com/_/microsoft-mssql-server), so it should be straightforward to test and eventually move to CI.

Thanks in advance!

**Comment by tingwei628** [comment]
Hi @imiric, I've already fixed the conflict! How do i add test.js ?

**Comment by imiric** [comment]
Thanks for the quick response!

1. Similar to the existing [`test.js`](https://github.com/imiric/xk6-sql/blob/fc5bb4f4a44cf893c2dd24c68516812316ddb4e8/test.js) for SQLite, but using MSSQL syntax. And moving both files to a `tests` subdirectory. I'll add a Postgres one and maybe hook up some integration tests in CI.

2. I removed it recently in f43fcf1.

**Comment by tingwei628** [comment]
Hi @imiric, I've added `tests/` in `1678c4a` and rename original test.js into `sqlite3_test.js`
Thanks !

File: tests/mssql_test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import sql from 'k6/x/sql';
+
+const db = sql.open("sqlserver", "./test.db");
```
Comment (imiric) [inline tests/mssql_test.js:3]: The second argument should be a connection string to showcase how it's specified for MS SQL, but it's OK, I'll fix it when I add the CI tests.

**Review by imiric — APPROVED** [review]
Thanks, I tested it with the SQL Server Docker image, and it works fine after I specified the connection string correctly.

So I'll merge this, and make some more changes, like upgrading `go-mssqldb` to the latest version, since 0.10.0 was released recently.

**Pull request merged**